.circleci/config.yml

```diff
@@ -602,3 +608,23 @@ workflows:
                 - master
     jobs:
       - uat_chain_snapshot
+
+  nightly_fast_integration_deploy:
+    triggers:
+      - schedule:
+          cron: "0 23 * * *"
+          filters:
+            branches:
+              only:
+                - master
+    jobs:
+      - deploy_fast_integration:
+          requires:
+            - test
```

Comment:
Do these jobs get triggered automatically overnight because of this dependency?

Reply:
Good question. I'll shall setup a mock

Reply:
Theoretically, `test`, `eunit`, `static_analysis` and `linux_package` run on all branches and `uat_tests` run on `master`, so it looks like that should be fine.

Reply:
I pushed a mock in 40145eb1 that shall run at 9:30 UTC (after ~10mins)

Reply:
```
We weren't able to start this workflow.
The following jobs are unreachable: deploy_fast_integration
At least one job in the workflow must have no dependencies.
deploy_fast_integration: requires job "static_analysis" but "static_analysis" is not part of this workflow.
deploy_fast_integration: requires job "eunit" but "eunit" is not part of this workflow.
deploy_fast_integration: requires job "linux_package" but "linux_package" is not part of this workflow.
deploy_fast_integration: requires job "test" but "test" is not part of this workflow.
```

Reply:
@lucafavatella I pushed new mock 9be03d24

I guess we should just assume the master is in good shape and depend only on `linux_package`

Reply:
https://circleci.com/workflow-run/7b17470a-4373-4f19-b68c-797cf02fde98

Reply:
Passed. I prepared changes in 7e592f33.

I prefer to "risk" and rely on stable master than copy pasting all the jobs in this cron workflow.